src/__tests__/useToast.test.tsx

```diff
@@ -167,6 +176,29 @@ describe('test useToast hook', () => {
     expect(onHide).toHaveBeenCalled();
   });
 
+  it('automatically hides when autoHide: true and panning.current: true', () => {
```

Comment:
By reading what the test actually asserts, I think the test case description should be:
```suggestion
  it('does not hide when autoHide is true but user is panning', () => {
```

Reply:
https://github.com/calintamas/react-native-toast-message/pull/574/commits/7ce8e39fc80e71bc0fbe14a0741d67e28fc0f9ff